modules/core/shared/src/main/scala/gem/math/ProperMotion.scala

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package gem.math
+
+import cats.implicits._
+import scala.math.{ sin, cos, hypot, atan2 }
+
+/**
+ * Time-parameterized coordinates, based on an observed position at some point in time (called
+ * the `epoch`) and measured velocities in distance (`radialVelocity`; i.e., doppler shift) and
+ * position (`properVelocity`) per year. Given this information we can compute the position at any
+ * instant in time. The references below are ''extremely'' helpful, so do check them out if you're
+  * trying to understand the implementation.
+ * @see The pretty good [[https://en.wikipedia.org/wiki/Proper_motion wikipedia]] article
+ * @see Astronomical Almanac 1984 [[https://babel.hathitrust.org/cgi/pt?id=uc1.b3754036;view=1up;seq=141 p.B39]]
+ * @see Astronomy and Astrophysics 134 (1984) [[http://articles.adsabs.harvard.edu/cgi-bin/nph-iarticle_query?bibcode=1984A%26A...134....1L&db_key=AST&page_ind=0&data_type=GIF&type=SCREEN_VIEW&classic=YES p.1-6]]
+ * @param baseCoordinates observed coordinates at `epoch`
+ * @param epoch           time of the base observation; typically `Epoch.J2000`
+ * @param properVelocity  proper velocity '''per year''' in [[RightAscension]] and [[Declination]], if any
+ * @param radialVelocity  radial velocity (km/y, positive if receding), if any
+ * @param parallax        parallax, if any
+ */
+final case class ProperMotion(
+  baseCoordinates: Coordinates,
+  epoch:           Epoch,
+  properVelocity:  Option[Offset],
+  radialVelocity:  Option[Double], // for now
```

Comment:
I don't know if it's worth the trouble newtyping this since it's never used anywhere else. I guess with Libra or whatever we could do this without giving it much thought.